.eslintignore

```diff
@@ -1 +1,13 @@
-syntax_error_trigger.js
\ No newline at end of file
+syntax_error_trigger.js
+spec/component*/*
+spec-integration/integration_component/*
+lib/amqp.js
```

Comment:
It seems strange to me,
to enable eslint with new rules and in the next step
just to ignore all or almost all code of sailor